apollo-gradle-plugin/src/test/files/gradle/settings.gradle

```diff
@@ -3,7 +3,7 @@ rootProject.name = "testProject"
 dependencyResolutionManagement {
   versionCatalogs {
     libs {
-      from(files("../../../gradle/libs.versions.toml"))
+      from(files("../../../gradle/libraries.toml"))
```

Comment:
Because we can't add `golatac` before the `plugins {}` block is parsed, plugins can't use `golatac` and the Gradle tests are still using the generated accessors.